index.html

```diff
@@ -282,11 +282,10 @@
           Push service
         </h2>
         <p>
-          The term <dfn>push service</dfn> refers to an overall end-to-end system that allows
-          <a title="application server">application servers</a> to send <a title=
-          "push message">push messages</a> to a <a>webapp</a>. A push service has a server
-          reachable at an associated endpoint. Such servers typically expose APIs specific to the
-          <a>push service</a>, e.g. for <a>push message</a> delivery initiation.
+          The term <dfn>push service</dfn> refers to system that allows
+          <a title="application server">application servers</a> to send <a title= "push
+          message">push messages</a> to a <a>webapp</a>. A push service has a server reachable at
+          one or more <a>endpoint</a>s.
```

Comment:
I believe that the point of the original sentence was to indicate that the push service is the thing that serves the "endpoint".  One or more came out of the fact that I was moving toward #129.